tidy3d/components/data/data_array.py

```diff
@@ -133,9 +161,8 @@ def _interp_validator(self, field_name: str = None) -> None:
         This does not check every 'DataArray' by default. Instead, when required, this check can be
         called from a validator, as is the case with 'CustomMedium' and 'CustomFieldSource'.
         """
-        # skip this validator if currently tracing for autograd because
-        # self.values will be dtype('object') and not interpolatable
-        if self.tracers is not None:
+        # skip this validator if currently tracing for autograd
+        if self.has_tracers:
```

Comment:
why skip exactly? can't we interpolate tracer. containing DataArrays now?

Reply:
Yeah this probably should have been removed in the interp PR...